### PR TITLE
docs(web3): adjust funding splits

### PR DESCRIPTION
**Pull request by nya-elimu**
Adjust funding splits for the repo's top 10 contributors. Based on the Contributors Table at https://app.armitage.xyz/projects/00f59cd4-89ff-4f7a-9a33-7e826a7e40df

ref https://github.com/elimu-ai/web3-wiki/issues/31

### CODE REVIEW

**Comment by nya-elimu** [comment]
@venkatesh2k3 Please add your ETH address so that you can get included in the monthly funding splits at https://www.drips.network/app/projects/github/elimu-ai/webapp

File: FUNDING_SPLITS.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-0x4709ebf314c6492d57f4c6d4f57357d5b2bb074e,vrudas,0.56
-0x914dBEe36BcAb63B2D15fAED08839Be43797b421,watr-sheep,0.38
+0x5D388Ec24Cc2C0C77458338696aa63AFB706A7b1,nya-elimu,49.39
+0xd46Cc93E3eE6a4fb532D9B48E95be7eD8f8f1DA0,jo-elimu,39.00
+0x3965d420cde24fe00f742cb31257ca90e04556a7,tomaszsmy,1.14
+0x819e4F5E912abb36c8f6c03790e7508e03cc37a3,UmenR,1.08
+0x342FaeB14127A79735144bbdC772AbB6b1bBe343,coderabbitai,1.05
+0x5367a6d4F5991327Bb20466684790e394bE000dE,Shubham-Vishwakarma,0.73
+0x0000000000000000000000000000000000000000,aschworer,0.70
+0x0000000000000000000000000000000000000000,venkatesh2k3,0.58
```
Comment (nya-elimu) [inline FUNDING_SPLITS.csv:9]: @venkatesh2k3 ☝🏽

Comment (venkatesh2k3) [reply FUNDING_SPLITS.csv:9]: What is the ETH address? and where I get it.

Comment (venkatesh2k3) [inline FUNDING_SPLITS.csv:9]: 0x513DceE2929a6e0CD115fb65Ec926d6569d98aff

Comment (venkatesh2k3) [reply FUNDING_SPLITS.csv:9]: @nya-elimu Do I need to create new branch and add the ETH address?

Comment (nya-elimu) [reply FUNDING_SPLITS.csv:9]: > 0x513DceE2929a6e0CD115fb65Ec926d6569d98aff

@venkatesh2k3 Added it for you: https://github.com/elimu-ai/webapp/commit/eb36baa6805ef80b5a350950569b725c45335ff6

Comment (venkatesh2k3) [reply FUNDING_SPLITS.csv:9]: Thank you @nya-elimu

**Pull request merged**